### ISSUE TITLE
new sales order type

### ISSUE THREAD

**Issue by chanlung** [frappe/erpnext#7904]
It would seem that a Sales Order of type "website" or "online" or "ecommerce" is in order.  Where this is an external website that does the e-commerce experience (for example: woocommerce or another popular website platform).  The flow for such a sales order would be SO->DN.  No Sales Invoice(billing) is needed as the website platform would have collected payment already.  The Sales Order would need to enter the Payment Information (similar to Sales Invoice "isPos").

**Comment by vrms** [comment]
can you structure your issue a bit more clearly? It's hard to follow what you actually want
